test/functional/apps/dashboard_elements/controls/control_group_chaining.ts

```diff
@@ -107,12 +107,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('Selecting an option in the first Options List will filter the second and third controls', async () => {
       await dashboardControls.optionsListOpenPopover(controlIds[0]);
-      await dashboardControls.optionsListPopoverSelectOption('cat');
+      await dashboardControls.optionsListPopoverSelectOptions('cat');
```

Comment:
Because the logic of checking to ensure the appropriate elements are selected is **pretty** expensive / slow, I decided to reduce the amount we call this by allowing multiple selections to be made at once **before** the check is made; that way, when we previously had multiple consecutive `optionsListPopoverSelectOption` calls, we can now call `optionsListPopoverSelectOptions` with an array and the selections check will only run a single time.